src/main/java/org/openmainframeproject/cobolcheck/features/interpreter/Block.java

```diff
@@ -0,0 +1,25 @@
+package org.openmainframeproject.cobolcheck.features.interpreter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Block {
```

Comment:
Using a combined term for section and paragraph was only a suggestion. I feel that 'Block' is too vague and would prefer the more precise sectionOrParagraph. There also exists a COBOL term; `BLOCK CONTAINS`, which is related to the `FILE SECTION`, one might get that confused. I'm sorry to make you redo it again, but I prefer the class name you had before.